docs/docs/content/core/run.md

```diff
@@ -1,10 +1,10 @@
 ---
 title: 'run'
-kind: 'primitive'
+kind: 'core'
 description: 'run is used to run parser with provided input.'
 ---
 
-# run <Primitive />
```

Comment:
FYI: removed badges from `run` and `tryRun`, since these are neither "primitives" or "composites" according to their definitions.